Workshop ID#142 new Lab 13: "Public and Private Network Access in Autonomous Database"

greenrick opened the pull request:
Workshop ID#142 "Manage and Monitor Autonomous Database", new Lab 13: "Public and Private Network Access in Autonomous Database". 
This lab focuses on showing how to limit network access using private endpoints.

anooshapilli commented on shared/workshops/freetier-indepth-test-private-endpoints/intro.md:
```diff
@@ -0,0 +1,53 @@
+# Introduction
+
```
@greenrick - Add heading 2 - "About this Workshop" after the Introduction

anooshapilli commented on shared/workshops/freetier-indepth-test-private-endpoints/intro.md:
```diff
@@ -0,0 +1,53 @@
+
+Oracle’s Autonomous Database is the perfect quick-start service for fast data loading and sophisticated data reporting and analysis. Oracle manages everything for you so you can focus on your data.
+
+## Workshop Objectives
```
Remove the word Workshop before the Objectives and Prerequisites

anooshapilli commented on shared/workshops/freetier-indepth-test-private-endpoints/intro.md:
```diff
@@ -0,0 +1,53 @@
+
+Use these links to get more information about Oracle Autonomous Database:
+
+- <a href="https://www.oracle.com/database/autonomous-database.html" target="\_blank">Oracle Autonomous Database website</a>
```
@greenrick - Please use the markdown format to reference the links

anooshapilli reviewed (commented):
@greenrick - Please make the requested in-line changes to the workshop

greenrick commented:
@anooshapilli - I fixed the 3 issues you raised. Actually, you found them in a temporary testing folder that I should have deleted, and which I just now deleted: shared/workshops/freetier-indepth-test-private-endpoints/. However, I made the changes that you raised in the actual folders for production.

Workshops 141 and 142 fixed items pointed out by Anoosha Pilli from 1st pull request, mostly in intro.md files
- Added "About this Workshops" under "Introduction"
- Removed "Workshop" from "Workshop Objectives" and "Workshop Prerequisites"
- removed 2 temporary folders in which Anoosha found things needed fixing, but made those fixes in the actual production folders. Removed files from "\adb\workshops\freetier-indepth-test-private-endpoints" and entirely removed "freetier-indepth-test-create-clone-from-backup" (at the moment, there's a file open somewhere on my local PC that doesn't allow me to delete the empty \adb\workshops\freetier-indepth-test-private-endpoints folder.)
